dask/dataframe/tests/test_indexing.py

```diff
@@ -431,19 +431,19 @@ def test_getitem_timestamp_str():
     ddf = dd.from_pandas(df, 10)
 
     # partial string slice
-    assert_eq(df["2011-01-02"], ddf["2011-01-02"])
-    assert_eq(df["2011-01-02":"2011-01-10"], df["2011-01-02":"2011-01-10"])
+    assert_eq(df.loc["2011-01-02"], ddf.loc["2011-01-02"])
+    assert_eq(df.loc["2011-01-02":"2011-01-10"], df.loc["2011-01-02":"2011-01-10"])
```

Comment:
```suggestion
    assert_eq(df.loc["2011-01-02":"2011-01-10"], ddf.loc["2011-01-02":"2011-01-10"])
```

It's the same in the original test, but should this actually be `ddf` for the right dataframe?

Reply:
definitely yes. That's what I get for searching and replacing...

Reply:
No, no, it was already wrong before your change ;)

Reply:
Yep, but I had a chance to inprove things! :)